security/subsystem/src/main/java/org/jboss/as/security/SecurityDomainResourceDefinition.java

```diff
@@ -63,9 +64,12 @@
  */
 class SecurityDomainResourceDefinition extends SimpleResourceDefinition {
 
+    static final String INFINISPAN_CACHE_TYPE = "infinispan";
+    static final RuntimeCapability<Void> INFINISPAN = RuntimeCapability.Builder.of("org.wildfly.security.domain.infinispan", true).build();
```

Comment:
@bstansberry This is meant to be a private capability - as the endpoint of a reference to an infinispan cache container capability.